reactive/kubernetes_e2e.py

```diff
@@ -32,121 +32,132 @@
 from subprocess import check_output
 
 db = unitdata.kv()
-USER = 'system:e2e'
-certs_dir = Path('/srv/kubernetes')
-ca_crt_path = certs_dir / 'ca.crt'
+HTTP_RELATION = "kubernetes-master"  # wokeignore:rule=master
+APP = hookenv.application_name()
+USER = "system:e2e"
+certs_dir = Path("/srv/kubernetes")
+ca_crt_path = certs_dir / "ca.crt"
 
 
-@hook('upgrade-charm')
+@hook("upgrade-charm")
 def reset_delivery_states():
-    ''' Remove the state set when resources are unpacked. '''
+    """Remove the state set when resources are unpacked."""
     install_snaps()
 
 
-@when('kubernetes-e2e.installed')
+@when("kubernetes-e2e.installed")
 def report_status():
-    ''' Report the status of the charm. '''
+    """Report the status of the charm."""
     messaging()
 
 
 def messaging():
-    ''' Probe our relations to determine the proper messaging to the
-    end user '''
+    """Probe our relations to determine the proper messaging to the
+    end user"""
 
     missing_services = []
-    if not is_state('kubernetes-master.available'):
-        missing_services.append('kubernetes-master:http')
-    if not is_state('certificates.available'):
-        missing_services.append('certificates')
-    if not is_state('kubeconfig.ready'):
-        missing_services.append('kubernetes-master:kube-control')
+    if not is_state(f"{HTTP_RELATION}.available"):
+        missing_services.append(f"{APP}:{HTTP_RELATION}")
+    if not is_state("certificates.available"):
+        missing_services.append(f"{APP}:certificates")
+    if not is_state("kubeconfig.ready"):
+        missing_services.append(f"{APP}:kube-control")
```

Comment:
if no relations are present, `juju status` will change from:

```
Missing relations: kubernetes-master:http, certificates, kubernetes-master:kube-control
```

to

```
Missing relations: kubernetes-e2e:kubernetes-master, kubernetes-e2e:certificates, kubernetes-e2e:kube-control
```

Reply:
these reflect the three relations which need to be related to the control-plane charm and easyrsa